### PR TITLE
Final review of the content

### LINKED ISSUE #236: Larger subheadings in "Ecosystem"
I'd suggest enlarging the Ecosystem subheadings (Visualization, Data Science)... so they're less easy to overlook.

### LINKED ISSUE #237: Serial commas
There's precedent in the NumPy site for use of the serial comma -- that is, a comma before the final 'and' in a list of three things or more:

`dev/index.rst:240:` 
the documentation. Matplotlib, SciPy, and IPython are also required.

`dev/conduct/code_of_conduct.rst:37:`
2. Be empathetic, welcoming, friendly, and patient. We work together to resolve
<br>
The new site does not use it:

`./public/index.html:670:`
Several of these newer libraries such as CuPy, Sparse and Dask,

`./public/index.html:205:`
NumPy offers comprehensive mathematical functions, random number generators, linear algebra routines, Fourier transforms and more.

Based on precedent (and the [Google guide](https://developers.google.com/style/commas), if we adopt it), the new site should use serial commas too. 

I'd be willing to write the PR.

### PR DESCRIPTION
**Pull request by InessaPawson**
1. Fixed grammar errors.
2. Addressed #237 ( Closes #237 ).
3. Replaced “she” with singular “they”.
4. Checked all the links.

### CODE REVIEW

File: content/en/installing-python-and-numpy-guide.md [hunk 1]
```diff
@@ -3,40 +3,40 @@ title: Guide to installing Python and NumPy
 sidebar: false
 ---
 
-Installing and managing packages in Python land is complicated, there are a
+Installing and managing packages in the Python land is complicated, there are a
```
Comment (rgommers) [inline content/en/installing-python-and-numpy-guide.md:6]: this change looks incorrect

File: content/en/installing-python-and-numpy-guide.md [hunk 2]
```diff
@@ -70,56 +70,56 @@ Otherwise:
...
 
-The third difference is that Pip does not have a _dependency resolver_ (this is
-expecting to change next year though) while Conda does. For simple cases (e.g.
-you just want NumPy, SciPy, Matplotlib, Pandas, Scikit-learn and a few other
-packages) that doesn't matter, however for complicated cases Conda can be
-expected to do a better job keeping everything working well together.  The flip
-side of that coin is that installing with Pip is typically a _lot_ faster than
-installing with Conda.
+The third difference is that pip does not have a _dependency resolver_ (this is
+expecting to change in the near future), while conda does. For simple cases (e.g.
```
Comment (rgommers) [inline content/en/installing-python-and-numpy-guide.md:113]: caught another small issue here, should be "expected to change"

File: content/en/installing-python-and-numpy-guide.md [hunk 3]
```diff
@@ -158,33 +158,33 @@ importing it in notebooks).
 
 ## NumPy packages & accelerated linear algebra libraries
 
-NumPy doesn't depend on any other Python packages, however it does depend on an
+NumPy doesn't depend on any other Python packages, however, it does depend on an
 accelerated linear algebra library - typically
 [Intel MKL](https://software.intel.com/en-us/mkl) or
-[OpenBLAS](https://www.openblas.net/).  The user doesn't have to worry about
+[OpenBLAS](https://www.openblas.net/). Users doesn't have to worry about
```
Comment (rgommers) [inline content/en/installing-python-and-numpy-guide.md:164]: "Users don't ..."

File: content/en/installing-python-and-numpy-guide.md (hunk 1 above)
Comment (rgommers) [reply content/en/installing-python-and-numpy-guide.md:6]: since this apparently isn't common or too colloquial, I'll remove "land".

**Comment by rgommers** [comment]
Merged, thanks @InessaPawson

File: content/en/installing-python-and-numpy-guide.md (hunk 1 above)
Comment (InessaPawson) [reply content/en/installing-python-and-numpy-guide.md]: You could have said "the Python land", it does require a definite article.

File: content/en/installing-python-and-numpy-guide.md (hunk 3 above)
Comment (InessaPawson) [reply content/en/installing-python-and-numpy-guide.md]: I was focusing on the overall writing style of this page, and neglected the basics. This is what fatigue does to me.:( My apologies.

Comment (rgommers) [reply content/en/installing-python-and-numpy-guide.md]: no worries at all, that's what review is for:) 

> This is what fatigue does

same here

**Pull request merged**